src/cgame/cg_predict.cpp

```diff
@@ -243,7 +243,7 @@ CG_Trace
 ================
 */
 void CG_Trace( trace_t *result, const vec3_t start, const vec3_t mins, const vec3_t maxs,
-               const vec3_t end, int skipNumber, int mask, int skipmask )
+               const vec3_t end, int skipNumber, int mask, int skipmask, traceType_t tt )
```

Comment:
Why do you want to add a new parameter that's the same in every call? This is just making the code uglier.

Reply:
This sounds fine to me